float64 fl_side
float64 fr_side
float64 bl_side
float64 br_side
float64 l_front
float64 r_front